Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -902,7 +903,7 @@ open class NavigationMapView: UIView {
               let triangleImage = Bundle.mapboxNavigation.image(named: "triangle")?.withRenderingMode(.alwaysTemplate) else { return }
         
         do {
-            try mapView.mapboxMap.style.addImage(triangleImage, id: NavigationMapView.ImageIdentifier.arrowImage)
+            try mapView.mapboxMap.style.addImage(triangleImage, id: NavigationMapView.ImageIdentifier.arrowImage, stretchX: [], stretchY: [])
```

Comment:
mapbox/mapbox-maps-ios#720 proposes giving these parameters default arguments so they can be omitted again.